python/lsst/analysis/ap/apdbCassandra.py

```diff
@@ -95,6 +95,8 @@ def __init__(
             )
         self._apdb = apdb
 
+        # TODO: do we need to get instrument here?
```

Comment:
A great question. Perhaps @isullivan knows the answer?

Reply:
I'll make a note to discuss this at the Monday AP meeting, and wait to merge on that.